.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -20,3 +20,6 @@ labels: feature
 ### Additional Context
 <!-- Add any other information or screenshots about the feature request here. -->
 
+
+### (Optional) Design Requirements
```

Comment:
```suggestion
### Design Requirements
```
I guess it's simpler to just delete this section when there are no design requirements...